ansible/roles/azure/tasks/create_adlsgen2.yml

```diff
@@ -142,7 +144,7 @@
     api_version: '2015-07-01'
   register: RoleDefinitionInfo
 
-# Retrieve storage acount informationn. 
+# Retrieve storage acount informationn.
```

Comment:
Remove extra 'n' at the end.

Reply:
Good catch. I found a few more typos in this space which I will fix and push an update.